Arrowgene.Ddon.Shared/Entity/Structure/CDataAreaRankSeason3.cs

```diff
@@ -0,0 +1,30 @@
+using Arrowgene.Buffers;
+
+namespace Arrowgene.Ddon.Shared.Entity.Structure
+{
+    public class CDataAreaRankSeason3
+    {
+        public uint SpotId { get; set; }
+        public bool DeadlineReached { get; set; }
+        public ulong Deadline { get; set; } // Seemingly a DateTime?
```

Comment:
For datetimes we use the datetime type in the object and convert to ulong in the Serializer, you can find other examples out there